README.md

```
@@ -1,19 +1,102 @@
+- 🦾 **簡単なテーマカスタマイズ**
 
-・Prismaのアカウント登録
-https://console.prisma.io/login
+---
 
-・node.js v22以上のインストール（npm 10.5.1以上）
+## 📁 プロジェクト構成
 
-[2025/06/05]
-Redux導入
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)